echemdb/entry.py

```diff
@@ -99,17 +99,8 @@ def __dir__(self):
 
             >>> entry = Entry.create_examples()[0]
             >>> dir(entry) # doctest: +NORMALIZE_WHITESPACE
-            ['__class__', '__delattr__', '__dict__', '__dir__', '__doc__',
-            '__eq__', '__format__', '__ge__', '__getattr__', '__getattribute__',
-            '__getitem__', '__getstate__', '__gt__', '__hash__', '__init__', '__init_subclass__',
-            '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__',
-            '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__',
-            '__subclasshook__', '__weakref__',
-            '_descriptor', '_digitize_example',
-            'bibliography', 'citation', 'create_examples', 'curation',
-            'data_description', 'df', 'experimental', 'field_unit', 'figure_description',
-            'identifier', 'package',  'plot', 'profile', 'rescale', 'resources', 'source',
-            'system', 'version', 'yaml']
+            ['...
```

Comment:
Somehow this seems to behave differently with different python versions...

Reply:
In what sense? Does this work?

```suggestion
            [..., 'bibliography', 'citation', 'create_examples', 'curation',
            'data_description', 'df', 'experimental', 'field_unit', 'figure_description',
            'identifier', 'package',  'plot', 'profile', 'rescale', 'resources', 'source',
            'system', 'version', 'yaml']
```

With your change, the `NORMALIZE_WHITESPACE` could go away in any case.